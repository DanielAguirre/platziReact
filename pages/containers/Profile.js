import React, { Component } from 'react';
import { Link } from 'react-router-dom';

class Profile extends Component {
  render() {
    return (
      <section name="Home">
        <h1>Post</h1>
        <Link to="/">Go to Home</Link>
        <Link to="/random">Go to Random</Link>
      </section>
      )
  }
}

export default Profile;
