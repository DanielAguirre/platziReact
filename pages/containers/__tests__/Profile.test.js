import Profile from '../Profile';

it('Render the Profile componet', () => {
  const componet = shallow(<Profile/>);
  expect(componet).toMatchSnapshot();
});