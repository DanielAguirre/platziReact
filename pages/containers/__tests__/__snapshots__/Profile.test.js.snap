// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render the Profile componet 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <section
    name="Home"
>
    <h1>
        Post
    </h1>
    <Link
        replace={false}
        to="/"
    >
        Go to Home
    </Link>
    <Link
        replace={false}
        to="/random"
    >
        Go to Random
    </Link>
</section>,
  "nodes": Array [
    <section
      name="Home"
>
      <h1>
            Post
      </h1>
      <Link
            replace={false}
            to="/"
      >
            Go to Home
      </Link>
      <Link
            replace={false}
            to="/random"
      >
            Go to Random
      </Link>
</section>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance":  {
      "_calledComponentWillUnmount": false,
      "_compositeType": 0,
      "_context": Object {},
      "_currentElement": <Profile />,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": Profile {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {},
        "refs": Object {},
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <section
          name="Home"
>
          <h1>
                    Post
          </h1>
          <Link
                    replace={false}
                    to="/"
          >
                    Go to Home
          </Link>
          <Link
                    replace={false}
                    to="/random"
          >
                    Go to Random
          </Link>
</section>,
        "_debugID": 2,
        "_renderedOutput": <section
          name="Home"
>
          <h1>
                    Post
          </h1>
          <Link
                    replace={false}
                    to="/"
          >
                    Go to Home
          </Link>
          <Link
                    replace={false}
                    to="/random"
          >
                    Go to Random
          </Link>
</section>,
      },
      "_renderedNodeType": 0,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <Profile />,
}
`;
